src/gausskernel/storage/access/k2/storage.cpp

```diff
@@ -833,8 +833,8 @@ K2PgStatus serializePgAttributesToSKV(skv::http::dto::SKVRecordBuilder& builder,
 
 K2PgStatus makeSKVRecordFromK2PgAttributes(K2PgOid database_oid, K2PgOid table_oid,
                                            const std::vector<K2PgAttributeDef>& columns,
-                                           skv::http::dto::SKVRecord& record) {
-    std::shared_ptr<k2pg::PgTableDesc> pg_table = k2pg::pg_session->LoadTable(database_oid, table_oid);
+                                           skv::http::dto::SKVRecord& record,
+                                           std::shared_ptr<k2pg::PgTableDesc> pg_table) {
```

Comment:
Why add pg_table? Doesn't look like it is used

Reply:
It is used to find column by attr_num, see the following line,

        k2pg::PgColumn *pg_column = pg_table->FindColumn(column.attr_num);